### PR TITLE
[usecase] Automate 4 usecases

### PR DESCRIPTION
**Pull request by JianfengXu**
Impacted TCs num(approved): New 0, Update 4, Delete 0
Unit test Platform: Andriod
Unit test result summary: Pass 4, Fail 0, Blocked 0

### CODE REVIEW

File: usecase/usecase-webapi-xwalk-tests/testscripts/PerformanceTimeline.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Feature: Usecase WebAPI
+ Scenario: Performance & Optimization/WebStorage Test
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/testscripts/PerformanceTimeline.feature]: WebStorage? A bit mistake.

File: usecase/usecase-webapi-xwalk-tests/testscripts/PerformanceTimeline.feature [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    When launch "usecase-webapi-xwalk-tests"
+     And I go to "/samples/PerformanceTimeline/index.html"
+     And I press "pc1_send"
+     And I wait 20 seconds
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/testscripts/PerformanceTimeline.feature]: Why need wait so much time? Reduce it if possible to save the execution time.

File: usecase/usecase-webapi-xwalk-tests/testscripts/Workers.feature [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Feature: Usecase WebAPI
+ Scenario: Performance & Optimization/Usertiming Test
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/testscripts/Workers.feature]: Usertiming?

**Comment by Honry** [comment]
why no tests.xml update?

**Comment by JianfengXu** [comment]
Due to xml file need be updated to suit testkit-lite 3.1.14, I will add these four usecas to xml in next PR.

**Pull request merged**